src/components/nav/nav.js

```diff
@@ -1,25 +1,102 @@
 import React from 'react'
-import Link from 'gatsby-link'
+import Link from '../link/link';
+import { TiThMenu, TiTimes } from 'react-icons/lib/ti'
 import styled from 'styled-components'
 
 import { colors } from '../../colors'
+import './nav.scss'
 
 import Button from '../button/button'
 import logo from '../../../images/logo.svg'
+import { NavLink } from 'react-router-dom';
 
-const Nav = () => (
-  <StyledNav>
-    <NavInner>
-      {/** Menu to go here when we have more pages **/}
-      <Logo src={logo} />
-      <Brand>DECOUPLED DRUPAL DAYS</Brand>
-      { /** <RegisterButton>Register now</RegisterButton> **/}
-    </NavInner>
-  </StyledNav>
-)
+const Menu = (props) => {
+  return (
+  <ul className={"nav-menu " + (props.show ? "menu-show" : "menu-hide")}>
+    <li>
+      <NavLink
+        to="/about"
+        activeClassName="selected"
+      >about</NavLink>
+    </li>
+    <li>
+      <NavLink
+        to="/venue"
+        activeClassName="selected"
+      >venue</NavLink>
+    </li>
+    <li>
+      <NavLink
+        to="/volunteer"
+        activeClassName="selected"
+      >volunteer</NavLink>
+    </li>
+    <li>
+      <NavLink
+        to="/diversity-inclusion"
+        activeClassName="selected"
+      >diversity and inclusion</NavLink>
+    </li>
+  </ul>
+)};
+
+class Nav extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      showMenu: false,
+    }
+  }
+
+  render() {
+    return (
+    <StyledNav>
+      <NavInner>
+        <NavLinks>
+          <Toggle role="button" onClick={() => this.setState({showMenu: !this.state.showMenu})}>
+            {!this.state.showMenu ? (<TiThMenu />) : (<TiTimes />)}
+          </Toggle>
+          <Menu show={this.state.showMenu} />
+        </NavLinks>
+        <Link to="/">
+          <Logo src={logo} />
+        </Link>
+        <Brand>DECOUPLED DRUPAL DAYS</Brand>
+        { /** <RegisterButton>Register now</RegisterButton> **/}
```

Comment:
What is this about?

Reply:
That's a good point,, Just updated it to include the button.. I also updated the menu to be it's own file.